native/cocos/engine/Engine.cpp

```diff
@@ -362,8 +365,10 @@ bool Engine::dispatchWindowEvent(const WindowEvent &ev) {
     } else if (ev.type == WindowEvent::Type::SIZE_CHANGED ||
                ev.type == WindowEvent::Type::RESIZED) {
         cc::EventDispatcher::dispatchResizeEvent(ev.width, ev.height);
-        auto *w = CC_GET_PLATFORM_INTERFACE(ISystemWindow);
-        w->setViewSize(ev.width, ev.height);
+        auto *w = CC_GET_SYSTEM_WINDOW(ev.windowId);
+        if (w) {
```

Comment:
Window `w` might be empty?

Reply:
For safety

Reply:
We might use CC_ASSERT to ensure w must be non-null. You can make the choice here.